lib/objects/app_state.dart

```diff
@@ -25,6 +25,7 @@ class AppState extends ChangeNotifier {
   final FirebaseFirestore firestore;
 
   Future<void> _init() async {
+    isLoading = true;
```

Comment:
I don't think you want isLoading = true to be here. Only in the initialization of the variable. When this is run, it will set it false, that is what makes it able to distinguish between Loading and No Items Today